app/controllers/groups_controller.rb

```diff
@@ -736,6 +736,12 @@ def group_params(automatic: false)
       end
     end
 
+    Group.plugin_permitted_params.each do |key, value|
```

Comment:
I'm not sure how `DiscoursePluginRegistry` works but we seem to have a pattern for adding more permitted params. Can we see if we can extend that instead? 

On line `725`, I see